config/seata-config-zk/pom.xml

```diff
@@ -33,14 +33,8 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>com.101tec</groupId>
```

Comment:
我希望我们能保留zkclient实现一段时间，提供两个zk的configuration和registry的实现，以便用户平滑迁移
I hope we can keep the zkclient implementation for a while, provide the configuration of both zk and the implementation of the registry, so that users can migrate smoothly

Reply:
However, regardless of the situation, users still need to manually import the dependency once and cannot do it seamlessly unless we include this dependency.
但是无论如何，用户都要手动引入一次依赖，无法平滑，除非我们将这个依赖带上

Reply:
> However, regardless of the situation, users still need to manually import the dependency once and cannot do it seamlessly unless we include this dependency. 但是无论如何，用户都要手动引入一次依赖，无法平滑，除非我们将这个依赖带上

我们可以改为provided，然后加一些异常输出提示？
Can we change it to provided and add some exception output hints?